.travis.yml

```diff
@@ -4,8 +4,13 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
+matrix:
+  allow_failures:
+    - php: 7.0
```

Comment:
I would allow PHP 7 failures, it passes anyway https://travis-ci.org/KnpLabs/KnpPaginatorBundle/jobs/73402865